<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wDockOfflineMessages</class>
 <widget class="QWidget" name="wDockOfflineMessages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="twOfflineMessages">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string comment="LBL_SUBJECT">Subject</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="LBL_VISITOR">Visitors</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="LBL_DATE">Date</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbMessageView">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string comment="LBL_VIEW">View</string>
       </property>
       <property name="icon">
        <iconset resource="lh3.qrc">
         <normaloff>:/lh3/Resources/read_messages.png</normaloff>:/lh3/Resources/read_messages.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbMessageDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string comment="LBL_DELETE">Delete</string>
       </property>
       <property name="icon">
        <iconset resource="lh3.qrc">
         <normaloff>:/lh3/Resources/deleting_message.png</normaloff>:/lh3/Resources/deleting_message.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lh3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>twOfflineMessages</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>wDockOfflineMessages</receiver>
   <slot>itemDoubleClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twOfflineMessages</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>wDockOfflineMessages</receiver>
   <slot>itemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbMessageView</sender>
   <signal>clicked()</signal>
   <receiver>wDockOfflineMessages</receiver>
   <slot>ViewRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbMessageDelete</sender>
   <signal>clicked()</signal>
   <receiver>wDockOfflineMessages</receiver>
   <slot>DeleteRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>itemDoubleClicked(QTreeWidgetItem*,int)</slot>
  <slot>itemSelectionChanged()</slot>
  <slot>ViewRecord()</slot>
  <slot>DeleteRecord()</slot>
 </slots>
</ui>
